source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -53,20 +55,20 @@ To be chosen to bake a block, the baker must take part in a
 
 The same stake is used when calculating whether a baker is included in the :ref:`finalization <glossary-finalization>` committee or not.
 
+If a baker misses baking a block, it is known which baker missed it. This is useful information for delegators when choosing a baker pool to delegate to, and node runners.
+
 Time concepts
 -------------
-The Concordium blockchain divides time into :ref:`epochs <glossary-epoch>` and :ref:`slots <glossary-slot>`.
-
-.. image:: ../images/concepts/epochs-slots.png
-   :alt: timeline showing how 14400 slots make up one epoch
+The Concordium blockchain divides time into :ref:`epochs <glossary-epoch>` and :ref:`rounds <glossary-round>`.
```

Comment:
That is not completely correct. Epochs are a unit of time, but rounds are not. A round is just an index at which there can be a block or a timeout, but there is no amount of time associated with a round.